### PR TITLE
Get rid of the api creds env variable for Cloudsmith.

### PR DESCRIPTION
**Pull request by aressem**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by glebashnik** [comment]
Looks good.

**Comment by aressem** [comment]
Need approval before merge. Please approve the PR (Files changed->Review changes->Approve)

**Review by ean — APPROVED** [review]

**Pull request merged**
